Dockerfile

```diff
@@ -24,6 +24,10 @@ ARG PRODUCT_VERSION="2020.2.3"
 
 ARG BASE_MOUNT_FOLDER="/JetBrains"
 
+#Set JAVA_HOME
+ENV JAVA_HOME /usr/lib/jvm/java-11-openjdk/
+RUN export JAVA_HOME
```

Comment:
hello, AFAIK this instruction is useless as it's lost at the next layer

Reply:
@benoitf yes, `RUN export JAVA_HOME` is not required, will remove it . Thanks